license.md

```diff
@@ -1,110 +1,99 @@
+In order to receive this license, you have to agree to its rules.  Those rules are both obligations under that agreement and conditions to your license.  Don't do anything with this software that triggers a rule you can't or won't follow.
 
-<!-- Because the license puts significant obligations on the licensee, we need to dispel confusion about legal characterization and remedies. To head off wasteful arguments in uncharted legal territory, expressly recite both contract and license.  The terms alone cannot establish contract. But in the likely event that facts do, clarify the consequences. -->
-
-<!-- The final sentence goes to impossibility and frustration.  It may not be necessary, but we'd all better avoid the argument. -->
-
-## Copyright
-
-Each contributor licenses you to do everything with this software that would otherwise infringe that contributor's copyright in it.
-
-<!-- Identical to the Copyright section of the Blue Oak Model License 1.0.0 -->
+<!-- Identical to the Agreement section of Parity 7.0.0 -->
 
 ## Notices
 
-You must ensure that everyone who gets a copy of any part of this software from you, with or without changes, also gets the text of this license or a link to <https://apicopyleft.com/license/{version}>.
-
-<!-- Derived from to the Copyright section of the Blue Oak Model License 1.0.0 -->
-
-## Patent
+Make sure everyone who gets a copy of any part of this software from you, with or without changes, also gets the text of this license or a link to <https://apicopyleft.com/license/{version}>.
 
-Each contributor licenses you to do everything with this software that would otherwise infringe any patent claims they can license or become able to license.
+<!-- Based on the Notices section of Parity 7.0.0 -->
 
-<!-- Identical to the Patent section of the Blue Oak Model License 1.0.0 -->
+## Copyleft
 
-<!-- Note that Contributing requires a strong patent grant. -->
+[Contribute](#contribute) all changes and additions to this software, as well as all software that invokes this software's functionality.  When in doubt, [contribute](#contribute).
 
-## Reliability
+<!-- Based on the Copyleft section of Parity 7.0.0 -->
 
-No contributor can revoke this license.
+This rule requires cooperative development of this software, minimizing duplication of effort across competing substitutes.
 
-<!-- Identical to the Reliability section of the Blue Oak Model License 1.0.0 -->
+## Applications
 
-<!-- Express irrevocability.  Note that headings are _not_ disclaimed, and Reliance evokes the relevant legal concept. -->
+You don't have to contribute any software that only invokes this software's functionality through the interfaces this software exposes without exposing this software's interfaces or functionality to users or other software to such an extent that it becomes a practical substitute for this software.
```

Comment:
Following your previous [2.0.0-pre.2](https://apicopyleft.com/versions/2.0.0-pre.2.html#applications) version there is a comma missing here. 

>interfaces this software exposes, without exposing this